README.md

```diff
@@ -83,6 +83,20 @@ helm install --namespace=observe observe-traces observe/traces \
 	--set observe.token.value="${OBSERVE_TOKEN}"
 ```
 
+When upgrading the `traces` chart:
+
+```
+helm upgrade --namespace=observe observe-traces observe/traces --reuse-values \
+	--set global.observe.otelPath="/v2/otel"
+```
+
+When upgrading the `stack` chart:
```

Comment:
Does the otelPath parameter have any effect in the stack chart? I thought it's only limited the traces chart.

Reply:
Yes, users can install traces as a part of stack. In that case, they need to update stack chart